pandas/tests/series/test_operators.py

```diff
@@ -976,16 +976,27 @@ def run_ops(ops, get_ser, test_ser):
         # ## timedelta64 ###
         td1 = Series([timedelta(minutes=5, seconds=3)] * 3)
         td1.iloc[2] = np.nan
-        td2 = timedelta(minutes=5, seconds=4)
-        ops = ['__mul__', '__floordiv__', '__pow__', '__rmul__',
-               '__rfloordiv__', '__rpow__']
-        run_ops(ops, td1, td2)
-        td1 + td2
-        td2 + td1
-        td1 - td2
-        td2 - td1
-        td1 / td2
-        td2 / td1
+        tdscalar = Timedelta(minutes=5, seconds=4)
+        ops = ['__mul__', '__pow__', '__rmul__', '__rpow__']
+        run_ops(ops, td1, tdscalar)
+        td1 + tdscalar
+        tdscalar + td1
+        td1 - tdscalar
+        tdscalar - td1
+        td1 / tdscalar
+        tdscalar / td1
+        tm.assert_series_equal(td1 // tdscalar, Series([0, 0, np.nan]))
+        tm.assert_series_equal(td1 // tdscalar.to_pytimedelta(),
+                               Series([0, 0, np.nan]))
+        tm.assert_series_equal(td1 // tdscalar.to_timedelta64(),
+                               Series([0, 0, np.nan]))
+        # TODO: the Timedelta // td1 fails because of a bug
+        # in Timedelta.__floordiv__, see GH#18824
+        # tm.assert_series_equal(tdscalar // td1, Series([1, 1, np.nan]))
```

Comment:
I would consider breaking out into a separate test so we can `xfail` it separately.

Reply:
Works for me.  I've got a fix ready pending feedback on #18846.

Reply:
yes would clean up these tests here (IIRC you broke these out by operator elsewhere)